modules/osdk-cli-reference-generate-packagemanifests.adoc

```diff
@@ -0,0 +1,60 @@
+// Module included in the following assemblies:
+//
+// * operators/operator_sdk/osdk-cli-reference.adoc
+
+[id="osdk-cli-reference-generate-packagemanifests_{context}"]
+= packagemanifests
+
+Running `generate packagemanifests` subcommand is the first step to publishing your Operator to a catalog and/or deploying
```

Comment:
I believe that I overlooked this earlier, apologies for the careless error.  Please revise to avoid the slash, as indicated in the ISG under the "Slashes" topic.  My obligatory suggestion:

"Running the `generate packagemanifests` subcommand is the first step to publishing your Operator to a catalog, deploying it with OLM, or both."

However, any revision that avoids the "and/or" combination is good.  Apologies again if I missed this earlier.

Reply:
Yes. Looks like a miss from my side.